client/test_config_bundle.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Perhaps these can be moved to a `tests` directory that can be used later for unit tests? Also leaving this here to remind you to remove them from the main directory before merging.
Similarly with the other test files.

Reply:
I've created a `test_json` folder, but just to reiterate, I'm aiming to keep these even after merging this PR, because they'll be useful in testing future PRs which will be closely related to this one.